virl-base-images/vyos/readme.md

```diff
@@ -0,0 +1,77 @@
+This works for CML v2.7.2.
+
+The Hypervisor used to create the `.qcow2` is Proxmox.
```

Comment:
While I am a Proxmox user and like it a lot, Proxmox is not [yet] a supported CML platform, and many users will not have it.  I appreciate the detailed instructions here, but could you also include `virt-install` [instructions](https://docs.vyos.io/en/equuleus/installation/virtual/libvirt.html)?

Reply:
Added 
https://github.com/conceptual-perception/cml-community/blob/master/virl-base-images/vyos/virt-install-instructions.md